dags/map_reproducibility/utils.py

```diff
@@ -127,21 +128,97 @@ def copy_bucket_cmds():
 def get_metrics_cmds():
   # TODO(gunjanj007): get these parameters from the recipe
   get_metrics = (
+      "METRICS_FILE=$COMPLETE_JOB_NAME/metrics.txt",
       "python3 process_training_results.py --file"
       " dllogger.json --batch_size 2048 "
       "--num_accelerators 256 "
       "--precision fp8  "
       "--model_type gpt3-175b "
```

Comment:
We currently manually set the parameters for post-processing values. Are these same parameters also used in the Helm template for starting the job? If so, could we define them as variables to avoid repetition and share them between both commands

Reply:
They are not used in helm template and we will get rid of these params once we onboard to aotc recipes